installers/rpm-code/codes/flash.sh

```diff
@@ -3,54 +3,51 @@
 flash_main() {
     codes_dependencies hypre
     codes_download_proprietary "flash/FLASH-4.6.2.tar.gz" "FLASH4.6.2"
-    local flash_src_root=$PWD
-    patch_makefile
-    for n in CapLaser RTFlame
+    flash_patch_makefile
+    install -d 755 "${codes_dir[share]}"/flash4
+    for n in CapLaserBELLA RTFlame
     do
-        setup_$n $n
-        make_and_install $n
+        flash_make_and_install_type $n
     done
 }
 
-make_and_install() {
-    cd $1
+flash_make_and_install_type() {
+    "flash_setup_$n" "$n"
+    cd "$1"
     codes_make
-    install -m 755 flash4 "${codes_dir[bin]}"/flash4-$1
-    install -d 755 "${codes_dir[share]}"/flash4
-    install -m 444 setup_units "${codes_dir[share]}"/flash4/setup_units-$1
+    # POSIT: Sirepo assumes the exe is named flash4-flashType
```

Comment:
`# POSIT: sirepo.sim_data.flash.SimData.flash_exe_path assumes flash4-flashType`

Reply:
I'll get the right verbosity one of these days... :)

https://github.com/radiasoft/download/pull/95/commits/4f394e009f65fb6649b1f4a0d9103ddd9be63bde